python/taichi/main.py

```diff
@@ -401,7 +416,9 @@ def update(self, arguments: list = sys.argv[2:]):
     @register
     def format(self, arguments: list = sys.argv[2:]):
         """Reformat modified source files"""
-        parser = argparse.ArgumentParser(description=f"{self.format.__doc__}")
+        parser = argparse.ArgumentParser(
+            prog='ti format',
+            description=f"{self.format.__doc__}")
```

Comment:
@rexwangcc 
Before:
```
usage: tt [-h] [-d DIFF]
```
After:
```
usage: ti format [-h] [-d DIFF]
```

Reply:
Nice! In the meantime this also reduces the confusion for people who run `python -m taichi example`:

Before:
```
python -m taichi example
# usage: __main__.py [-h]
```

After:
```
python -m taichi example
# usage: ti example [-h]
```